modules/user-access/pages/user-credentials.adoc

```diff
@@ -72,33 +72,40 @@ These operations include:
 
 == Secrets
 
-Secrets are unique strings that serve as a user's credentials in certain circumstances. A user can have multiple secret strings. Each secret is associated with one user and their role for one graph. If the role is revoked, the secret also becomes invalid.
+Secrets are unique strings that serve as a user’s credentials in certain circumstances. A user can have multiple secret strings. Each secret binds to the user who creates it only and inherits all permissions and privileges from the owner. Starting from 4.1.0, secret is no longer bound to graphs and tokens.
 
 === Create a secret
-
+Use GSQL Command:
 [source,gsql]
 ----
 CREATE SECRET [<alias>]
 ----
-
 Use the `CREATE SECRET` command to generate a secret for the current user and graph. It is optional to provide an alias for the secret.
 
 The system will generate a random alias for the secret if the user does not provide an alias for that secret.
 Randomly generated aliases begin with `AUTO_GENERATED_ALIAS_` and include a random 7-character string.
 
-==== Required privilege
-
-At least one of the database access privileges on the given graph: `READ_DATA`, `CREATE_DATA`, `UPDATE_DATA`, `DELETE_DATA` or `EXECUTE_LOADINGJOB`. The built-in `queryreader` role satisfies this requirement.
+Use GSQL Rest API:
+[source,gsql]
+----
+POST /gsql/v1/secrets?userName=<user>&value=<secret content>&alias=<alias>
+----
+Use the `POST method to call secrets API` to create secrets.
 
+* Case 1: `userName` is optional, if `userName` is not specified, it creates secret for default user who logins.
+* Case 2: `alias` is optional, if `alias` is not specified, randomly generated aliases begin with AUTO_GENERATED_ALIAS_ and include a random 7-character string.
+* Case 3: `value` is optional, if `value` is not specified, randomly generated 32-character string.
```

Comment:
Same as above.
"Case 3: The value parameter is optional. If not specified, a random 32-character string will be generated."